src/AspNet.Security.OAuth.Twitch/TwitchAuthenticationHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://Twitch.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using AspNet.Security.OAuth.Extensions;
+using Microsoft.AspNet.Authentication;
+using Microsoft.AspNet.Authentication.OAuth;
+using Microsoft.AspNet.Http.Authentication;
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Twitch {
+    public class TwitchAuthenticationHandler : OAuthAuthenticationHandler<TwitchAuthenticationOptions> {
+        public TwitchAuthenticationHandler([NotNull] HttpClient client)
+            : base(client) {
+        }
+
+        protected override async Task<AuthenticationTicket> CreateTicketAsync([NotNull] ClaimsIdentity identity,
+            [NotNull] AuthenticationProperties properties, [NotNull] OAuthTokenResponse tokens) {
+            using (var request = new HttpRequestMessage(HttpMethod.Get, $"{Options.UserInformationEndpoint}?oauth_token={tokens.AccessToken}")) {
```

Comment:
Looks like Twitch supports the `Authorization` header: https://github.com/justintv/Twitch-API/blob/master/authentication.md#authenticated-api-requests. Using this header is preferable (mainly to avoid proxy caching). Could you please use it?